feat: add `user_name` to `ComputeSession` query field (#1149)
lablup/backend.ai: changes/1149.feature.md

> Add `user_name` to query session list with username.

Reviewer: For clarity: 
```suggestion
Expose `user_name` field to `compute_session` (and also `compute_session_list`) GQL query.
```

Author: I reflected it. Thank you.
